torch/onnx/_internal/exporter.py

```diff
@@ -799,9 +799,9 @@ def model_signature(self) -> Optional[torch.export.ExportGraphSignature]:
                               target='my_buffer2', persistent=True),
                     InputSpec(kind=<InputKind.BUFFER: 3>, arg=TensorArgument(name='arg5_1'),
                               target='my_buffer1', persistent=True),
-                    InputSpec(kind=<InputKind.USER_INPUT: 1>, arg=TensorArgument(name='l_x_'),
+                    InputSpec(kind=<InputKind.USER_INPUT: 1>, arg=TensorArgument(name='arg6_1'),
```

Comment:
Isnt it possible to preserve the name instead?

Reply:
trickier with the current approach. It also was not guaranteed that these names would look the way they do if strict=False. I plan to followup with a more comprehensive input processing pass that should name these more nicely.

Reply:
> trickier with the current approach. It also was not guaranteed that these names would look the way they do if strict=False. I plan to followup with a more comprehensive input processing pass that should name these more nicely.

is there an issue filed for this? would love to know more about it

Reply:
No, was just the conversation on this PR. Feel free to file an issue which we will keep updated

Reply:
https://github.com/pytorch/pytorch/issues/119484 thanks